### DISCUSSION TITLE
Create a compatible repository for Fedora/RHEL and openSUSE

### DISCUSSION THREAD

**Discussion by birdup000** [Floorp-Projects/Floorp#989] [Feature Request]
I enjoy Floorp very much and the only thing keeping me back from really wanting to use it is the clunkiness of Floorp on Flatpak since it is sluggish through there. Every once in a while it just freezes and kind of blanks out , I think creating a repo for these distributions will be awesome and help make it more widespread. A very large thank you to everyone behind this repo!

**Comment by CutterKnife** [comment]
The Flatpak version is managed by a third party. In the same way, if contributors could create packages for specific distributions, they would be able to provide them.

**Comment by SmollClover** [comment]
Not sure how relevant this really is anymore, but I personally would love an official RPM repository which would be compatible with Fedora since that is my daily driver.

Currently I'm using a third-party copr repository to get floorp through my package manager (dnf) due to the fact that I don't like having certain apps, such as web browsers, in a sandboxed environment such as flatpak since it often brings issues with it, and I also don't want to manually download a tar file every time there's an update to floorp. Instead having a comfortable way of just upgrading every package through dnf and getting an update to floorp is quite convenient and, to me at least, a way better experience.

I really do hope, despite how little this discussion got traction when it was created, that there will, at one point in the future, be an official RPM repository for floorp so I don't have to rely on a copr repository for it.

Oh, and in case anyone is curious and or wants to do the same, I am using the [sneexy/floorp](https://copr.fedorainfracloud.org/coprs/sneexy/floorp/) copr repository. So far no issues and, thanks to it automatically updating the spec files, I also quickly receive a update whenever floorp gets updated.

Also, here's the link to the git repository: https://git.gay/sneexy/copr/src/branch/main/floorp

**Comment by SnenxyTengoku** [comment]
bumping into here: i am the maintainer of the [sneexy/floorp](https://copr.fedorainfracloud.org/coprs/sneexy/floorp) copr repo, and i will say i'd prefer if there was an official, or at least better package available 

the one i provide has some minor issues (specifically, the application refuses to use its icon and i have no idea why) and i frankly don't know much about maintaining packages or managing spec files since this is just a fork of someone elses project with a daily script that automatically checks and replaces the version everyday.

basically, i'll still keep the package around and try my best to poke at it if something breaks but i'm not really "packager" material or anything

**Comment by ludgie** [comment]
Hello,

The package could be published on Terra, a third-party repo for Fedora and derivate distros.
https://terra.fyralabs.com/

Else, for it to get into Fedora repos it'd need for someone to apply as a Packager and then publish it, or ask someone to do it.
https://docs.fedoraproject.org/en-US/packaging-guidelines
https://docs.fedoraproject.org/en-US/package-maintainers/Joining_the_Package_Maintainers/

I've been trying to write a simpler spec than the one in COPR, but so far no luck. I get the error `Couldn't load XPCOM.` when trying to launch Floorp.
